go.mod

```diff
@@ -1,23 +1,22 @@
 module github.com/Layr-Labs/avs-sync
 
-go 1.21.13
+go 1.22.0
```

Comment:
is this bump actually needed for some 1.22 functionality? If so good to document why in the PR readme. This could break some clients who are importing avssync and using it directly.

Reply:
Looks like it was bumped to align with the version used in the go-ethereum `1.15.0` release. Have added to the readme

https://github.com/Layr-Labs/eigensdk-go/commit/dd1cd5bda45eccf9ad3d025cf71adf02b54628f8